fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -41,0 +1,131 @@
+"""Logging utilities."""
+import json
+import logging
+from datetime import datetime
+from logging import Logger
+from typing import Any, Dict, Optional
+
+
+class StructuredLogger:
+    """Custom logger that adds structured logging capabilities."""
+
+    def __init__(self, name: str | None = None, level: int = logging.DEBUG):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)